### PR TITLE
feat(Help Articles): Role based article display

### PR DESCRIPTION
**Pull request by wski**

### CODE REVIEW

**Comment by wski** [comment]
@clarkjefcoat

File: app/js/components/NavBar/helpmodal.jsx [hunk 1]
```diff
@@ -41,7 +59,7 @@ var HelpModal = React.createClass({
                         <h4 className="modal-title">{`${APP_TITLE} Help Zone`}</h4>
                         </div>
                         <div className="modal-body" style={{padding: '0px'}}>
-                            <iframe style={{width:"100%", height:"400px", border: 'none'}} src={HELP_URL} />
+                            <iframe style={{width:"100%", height:"400px", border: 'none'}} src={`${HELP_URL}/${this.state.role}`} />
```
Comment (clarkjefcoat) [inline app/js/components/NavBar/helpmodal.jsx]: This code depends on the HELP_URL not ending with a slash.  It does not work with the OzoneConfig.js that is currently checked-in to ozp-react-commons.  Chrome will not load the page http://localhost:8888/#//2

**Comment by clarkjefcoat** [comment]
Works as expected - other than the HELP_URL mismatch noted above.

**Comment by wski** [comment]
@clarkjefcoat

**Comment by clarkjefcoat** [comment]
:+1:

**Comment by wski** [comment]
@blaino

**Comment by blaino** [comment]
:+1:

**Pull request merged**